specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -733,6 +814,60 @@
           "readOnly": true
         }
       }
+    },
+    "AppTemplate": {
+      "description": "IoT Central Application Template.",
+      "type": "object",
+      "properties": {
+        "manifestId": {
```

Comment:
>manifestId [](start = 9, length = 10)

Is this supposed to be the identifier of the resource that's present in the GET route? If yes, then that's not allowed. (See my other comment.)

Reply:
it's not the identifier of the resource, and I changed the structure accordingly

Reply:
And we can not onboard get single template resource api now, we haven't implemented yet.

Reply:
Yes, you can. Once we approve this Swagger, you should hold off on asking for it to be merged. You just need signed off swagger PR to unblock your RP manifest change. Once you implement the service change to GET single template, you can ask to have the swagger PR merged to master.

---
In reply to: [271089778](https://github.com/Azure/azure-rest-api-specs/pull/5495#discussion_r271089778) [](ancestors = 271089778)